safe-global/safe-wallet-web
# [TX flow] Mobile optimisation

## What it solves

Resolves #2149 

## How this PR fixes it

- Moves the title up to the header area on smaller screens
- Adds a button to toggle the status widget on smaller screens

## How to test it

1. Open the Safe on a smaller screen < 900px
2. Create a new transaction
3. Observe the tx flow looks like the designs
4. Toggle the status widget on and off

## Screenshots
<img width="400" alt="Screenshot 2023-06-22 at 14 16 07" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/0ebb2f01-2504-49f5-b370-ebfe5b6ddb21">
<img width="393" alt="Screenshot 2023-06-22 at 14 16 23" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/a6aa4497-90ba-4d62-bdd5-69205bba9e98">

## Checklist
